src/components/Menu/Menu.tsx

```diff
@@ -90,24 +113,54 @@ const Menu = ({
     onClose && onClose();
   });
 
+  const handleSearch = (e: any) => {
```

Comment:
It would be nice to control whether it's a front-end search or back-end. For example, in my case I need to search entities